src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.figure;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract double getArea();
```

Comment:
let's instead create interface AreaCalculator

because we can calculate area of not only figures in our program later